platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiUserPortlet/Style.less

```diff
@@ -304,7 +304,7 @@ td.right-column-containerTDContainer {
 		padding: 20px 20px 20px 20px;
 		color: @textLightColor;
 		.findConnection {
-			padding-top: 40px;			
+			padding-top: 30%;
```

Comment:
The image height is defined in pixel (https://github.com/exodev/platform-ui/blob/c6231e8c1fc0a2a59be76586eca412423bc16a22/platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiUserPortlet/Style.less#L303), so we should do the same for this class IMO. We should set it to 65px.

Reply:
if we set the padding-top to 65px it will not be ok for the responsive as shown in the attached file.
i have 2 suggestions  : 
1 - leave it 30% which fixes the issue 
2 - keep it as it was 40px and in the mediascreen min width 1500px add .findConnection {
   padding-top: 65px; }
![screenshot responsive](https://user-images.githubusercontent.com/38003803/39200621-9054d0a2-47e4-11e8-8032-d690b7f7febe.png)

Reply:
> 1 - leave it 30% which fixes the issue
2 - keep it as it was 40px and in the mediascreen min width 1500px add .findConnection {
padding-top: 65px; }

It does not solve the issue when the text is always on 2 lines, like in german:
![selection_002](https://user-images.githubusercontent.com/342645/39241840-7dbe92a8-4888-11e8-8ccc-7d1943de3fb0.png)

Also, please create the PR on exodev since the issue occurs also in last PLF version.

Reply:
the text is in two lines even in english.(no connection text )(for media screen less than 1500px)
the problem was that the link "find connection" is misplaced as is shown in the attached file (i suggest to keep the word wrap for responsive)
![misplaced link](https://user-images.githubusercontent.com/38003803/39243483-7b92b69c-4885-11e8-8484-3ee53c5e18df.png)

Reply:
> the text is in two lines even in english.(no connection text )(for media screen less than 1500px)

I said "when the text is always on 2 lines", so even with media screen higher than 1500px.

> i suggest to keep the word wrap for responsive

Yes we must keep this behavior.

Reply:
for screenwidth >=  1500 px  no connection text is in 1 line  
for screenwidth < 1500px the text is wrapped as shown in attached files 
![media greater 1500](https://user-images.githubusercontent.com/38003803/39247856-6f70272a-4892-11e8-850d-9e781ac83410.png)
![scress less 1500](https://user-images.githubusercontent.com/38003803/39247858-73037928-4892-11e8-8428-a6dd4f64d2bc.png)
we can have a call to discuss more here is my skype : live:mohamed-omar.cherni